src/components/pages/photos/PhotosPage.js

```diff
@@ -5,10 +5,27 @@ import './PhotoPage.css';
 
 
 export default class PhotosPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentRover: 'Curiosity',
+    };
+  }
+
+  handleClick(roverName) {
+    this.setState({
+      currentRover: roverName,
+    });
+  }
+
   render() {
     return (
       <div className="photo-page">
-        <RoverNavbar />
+        <RoverNavbar
+          onClick={roverName => this.handleClick(roverName)}
+        />
+        {/*the li is for us to check clicking on the buttons does change the state */}
```

Comment:
We could probably put a TODO in this comment to remove the list once we implement the Rover component!